offer-manager-domain/src/main/groovy/life/qbic/business/products/Product.groovy

```diff
@@ -82,9 +111,13 @@ class Product {
 
         digest.update(product.description.getBytes(StandardCharsets.UTF_8))
 
+        digest.update(product.category.toString().getBytes(StandardCharsets.UTF_8))
+        digest.update(product.externalUnitPrice.toString().getBytes(StandardCharsets.UTF_8))
+        digest.update(product.serviceProvider.toString().getBytes(StandardCharsets.UTF_8))
+        digest.update(product.internalUnitPrice.toString().getBytes(StandardCharsets.UTF_8))
         digest.update(product.unit.value.getBytes(StandardCharsets.UTF_8))
         digest.update(product.unitPrice.toString().getBytes(StandardCharsets.UTF_8))
-        digest.update(product.category.toString().getBytes(StandardCharsets.UTF_8))
+
```

Comment:
why remove the category? Isnt it important to calculate prices?

Reply:
It is not removed I think

Reply:
ahhh ok found it. The highlighting was misleading